test/fixed/2.3.Lines.php

```diff
@@ -20,9 +20,9 @@ public function testLineLength(): void
     public function testTrailingWhitespace(): void
     {
         // There MUST NOT be trailing whitespace at the end of lines.
-        //
-        // Blank lines MAY be added to improve readability and to indicate
-        // related blocks of code except where explicitly forbidden.
+    //
```

Comment:
this seems weird though

Reply:
The latest updates from upstream broke the syntax. `2.3.Lines::testTrailingWhitespace`

Initially, I tried to get ci passing by adopting the new syntax. (to get familiar with the codebase)

Reply:
It still is a blocker, IMO: moving this means that we will break CI for all downstream projects

Reply:
I found out it's a bug in codesniffer. The rules are working correctly and no issues are reported. codesniffer fails to write the fixed code correctly. IMO, this should not be a blocker as long as we are aware of this and use a different comment style when this occurs.